src/views/day.php

```diff
@@ -16,6 +16,9 @@
 do_action( 'tribe_events_before_template' );
 ?>
 
+	<!-- Title Bar -->
+<?php tribe_get_template_part( 'day/titlebar' ); ?>
```

Comment:
Lets use `day/title-bar` for the name of this file.

Reply:
Fixed.